src/patient-lab-dashboard/patient-lab-details/patient-lab-details.tsx

```diff
@@ -183,36 +184,39 @@ const PatientLabDetails: React.FC<RouteComponentProps<PatientParamsType>> = ({
           </div>
           <br></br>
           <br></br>
-          <PendingLabOrdersProvider>
-            <PendingLabOrdersTable
+          <LabTestResultsProvider>
+            <PendingLabOrdersProvider>
+              <PendingLabOrdersTable
+                patientUuid={patientUuid}
+                onButtonClick={onButtonClick}
+                reloadTableData={reloadReportTable}
+              />
+              <br></br>
+              <br></br>
+              <Button renderIcon={AddFilled16} onClick={handleClick}>
+                Upload Report
+              </Button>
+              {onButtonClick && (
+                <UploadReportProvider>
+                  <UploadReport
+                    saveHandler={(isSaveSuccess = false) =>
+                      handleUploadAndSave(isSaveSuccess)
+                    }
+                    closeHandler={() => handleClose()}
+                    header="Upload Report"
+                    patientUuid={patientUuid}
+                  />
+                </UploadReportProvider>
+              )}
+              <br></br>
```

Comment:
It is generally advisable to having styling(margins) instead of adding line breaks. if there is a way to add space via margin please take that route